webpack/scenes/ContentViews/Details/Filters/CVRpmFilterContent.js

```diff
@@ -46,14 +51,14 @@ const CVRpmFilterContent = ({ filterId, inclusion }) => {
   const buildRows = useCallback((results) => {
     const newRows = [];
     results.forEach((rule) => {
-      const { name, architecture } = rule;
+      const { name, architecture, ...rest } = rule;
       const cells = [
         { title: name },
         { title: architecture || 'All architectures' },
         { title: versionText(rule) },
       ];
 
-      newRows.push({ cells });
+      newRows.push({ cells, ...rest });
```

Comment:
In similar usecases on other pages. we'd just push id to the row and use id below as rowData.id instead of { id } in the actionResolver..What do you think?

Reply:
I could optionally only pass in 'id' here, but below '{ id}'  is equivalent to rowData.id, and arguably better practice.